package.json

```
@@ -64,7 +65,8 @@
     "release-notes:lib": "ts-node ./scripts/changelog.ts --verbose --lib storefront",
     "release-notes:core": "ts-node ./scripts/changelog.ts --verbose --lib core",
     "release-notes:styles": "ts-node ./scripts/changelog.ts --verbose --lib styles",
-    "release-notes:schematics": "ts-node ./scripts/changelog.ts --verbose --lib schematics"
+    "release-notes:schematics": "ts-node ./scripts/changelog.ts --verbose --lib schematics",
+    "publish:docs": "gh-pages -d documentation -m 'Compodocs updates'"
```

Works great, apart from that message which is `'Compodocs` when using Windows shell. Maybe instead use `\"`? Could you check if this still works on mac?
```suggestion
    "publish:docs": "gh-pages -d documentation -m \"Compodocs updates\""
```

---

Still works on mac 👍